src/clib/lib/res_util/subst_list.cpp

```diff
@@ -359,8 +361,9 @@ char *subst_list_alloc_filtered_string(const subst_list_type *subst_list,
         }
 
         if (iterations >= max_iterations) {
-            throw std::runtime_error(
-                "Reached max iterations while trying to resolve defines");
+            throw std::runtime_error(fmt::format(
+                "Reached max iterations while trying to resolve defines in {}",
```

Comment:
Feel this sentence is not fully explaining what is going on - what about `what(): Reached max iterations while trying to resolve DEFINE for string <RMS_PROJECT>`